README.md

```diff
@@ -152,39 +152,39 @@ shown below. Can't see yours? Have a look at the [extra platform instructions](h
 
 1. Install the **starship** binary:
 
-   #### Install Latest Version
+#### Install Latest Version
 
-   ##### From prebuilt binary, with Shell:
```

Comment:
These indentations are intentional to make them sub-sections of the ordered list.
The rendered output is styled differently without the indentation.